io/post_regional.F90

```diff
@@ -829,6 +829,19 @@ subroutine set_postvars_regional(wrt_int_state,mpicomp,setvar_atmfile,   &
         enddo
       enddo
 !
+!** temporary fix: initialize t10m, t10avg, psfcavg, akhsavg, akmsavg
+!$omp parallel do default(none),private(i,j),shared(jsta_2l,jend_2u,im,spval), &
```

Comment:
`spval` is not used in the following do loop.

Reply:
I assume assigning 0 or spval have the same results in these fields. I have been working initializing all arrays in UPP code allocate_all.f. When that is done, this fix will be removed.

Reply:
fixed